entgql/template/mutation_input.tmpl

```diff
@@ -25,6 +25,9 @@
                 {{ print "Clear" $f.StructField }} bool
             {{- end }}
             {{ $f.StructField }} {{ if $f.IsPointer }}*{{ end }}{{ $f.Type }}
+            {{- if $f.AppendOp }}
+                {{ print "Append" $f.StructField }} {{ if $f.IsPointer }}*{{ end }}{{ $f.Type }}
```

Comment:
You can use `f.MutationAppend`: https://github.com/ent/ent/blob/master/entc/gen/type.go#L1122